process/dataAggregator.go

```diff
@@ -54,9 +62,14 @@ func (da *dataAggregator) ProcessHyperBlock(outportBlock *outport.OutportBlock)
 
 		log.Info("dataAggregator: get block", "hash", hash)
 
+		shardBlock, err := da.converter.HandleShardOutportBlock(outportBlockShard)
+		if err != nil {
```

Comment:
i think it's better to have convert outport block already in the pool

Reply:
that'd require several alterations. I'd rather keep it in a separate PR.